lib/tasks/update_resource_quantity.rake

```diff
@@ -0,0 +1,14 @@
+namespace :data do
+  desc 'Update resources with nil quantity'
+  task update_resource_quantity: :environment do
+    Resource.all.each do |resource|
```

Comment:
It makes sense to iterate only through the records that matter to us, instead of all the records.